### DISCUSSION TITLE
Is there any way to bypass .config when using flask Blueprint?

### DISCUSSION THREAD

**Discussion by JanusHasie** [pallets/flask#4354] [Q&A] [answered]
I am using flask Blueprint in my code but this does not allow for me to use .config, which causes a catch 22. I need to upload images from a web page to a database and this has really become quite a block.
Any advice?

**Comment by ThiefMaster** [comment, marked as answer]
Use `current_app.config`...

**Reply by JanusHasie** [reply]
Dude you saved my life!
